### ISSUE TITLE
USB not mounting NTFS

### ISSUE THREAD

**Issue by cmh714** [pimusicbox/pimusicbox#124]
My external USB HD with separate power wont auto-mount. Just looking for a quick fix...cant seem to figure out the correct mount point to get Mopidy to update its db.

tia

**Comment by blpp** [comment]
I downloaded the last official release 0.6 from the homepage. ntfs-3g was already installed. I followed the instructions from here http://www.techjawab.com/2013/06/how-to-setup-mount-auto-mount-usb-hard.html and added the following line to /etc/fstab.

`UUID=<MYID> /music/USB ntfs-3g uid=1001,gid=1002,umask=007 0 0`

The USB HDD with the NTFS partition is mounted at startup now and scanned by Mopidy.

I don't know whether it is possible to include a startup script to the musicbox image which is doing this automatically (checking for not already mounted drives, trying to mount them with NTFS).

**Comment by kingosticks** [comment]
As @woutervanwijk said,  automatic usb mounting is handled by the `usbmount` program which has a configurable list of filesystem types. I think the config file lives in /etc/usbmount so hopefully it's just a case of adding ntfs to that.

**Comment by blpp** [comment]
Ok, I've commented out my entry in the /etc/fstab. After that, I edited the file `/etc/usbmount/usbmount.conf`:

1.) I added the value ntfs to FILESYSTEMS:
`FILESYSTEMS="ntfs vfat ext2 ext3 ext4 hfsplus"`

2.) I added the options from the fstab file to FS_MOUNTOPTIONS:
`FS_MOUNTOPTIONS="-fstype=ntfs,uid=1001,gid=1002,umask=007"`

After restarting the muscibox the USB HDD with the NTFS file system was still found! So it seems that if these two lines are added a new version of the MusicBox with out of the box support of NTFS formatted USB devices can be released.
